llvm/lib/Target/X86/X86ISelLowering.cpp

```diff
@@ -56904,6 +56904,10 @@ X86TargetLowering::getRegForInlineAsmConstraint(const TargetRegisterInfo *TRI,
         if (!Subtarget.hasFP16())
           break;
         [[fallthrough]];
+      case MVT::v8bf16:
+        if (!Subtarget.hasBF16())
```

Comment:
Are multiple fallthroughs going to work? What happens for a subtarget that supports FP16 but not BF16?

Reply:
Good catch! Repalce fallthrough with return.